bin/build-app

```diff
@@ -5,5 +5,6 @@ set -o errexit
 
 python manage.py syncdb --noinput
 python manage.py migrate cabotapp --noinput
+python manage.py migrate statuscheckapp --noinput
```

Comment:
Yeah sure that makes sense--metricsapp? metricsstatuscheckapp is too long, not sure what else would be descriptive enough